packages/eas-cli/src/project/publish.ts

```diff
@@ -275,10 +276,14 @@ export async function uploadAssetsAsync(assetsForUpdateInfoGroup: CollectedAsset
     missingAssets.map(ma => ma.contentType)
   );
 
+  const assetUploadPromiseLimit = promiseLimit(50);
```

Comment:
I suspect the bottleneck will usually be the developer's upload bandwidth, not GCS's ability to receive data and store it to disk. In other works, I don't think a high-level of parallelization helps that much and a lower limit like 5? 10? 15? might be faster since there's less congestion.

Reply:
The only piece of concrete data I have is that it seemed like I was rate limited when I tried to upload 1500 assets at once (even on a fast connection with lots of upload bandwidth). 

I'll make this 15 though since it does make sense that a lower-level of parallelization makes sense.